### PR TITLE
feat: add more examples (#11)

### PR DESCRIPTION
**Pull request by vladisavvv**

### CODE REVIEW

File: examples/image-echo/app.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                return
+
+            image_base64 = base64.b64encode(image_bytes).decode("utf-8")
+
+            # Add content with the user image in base64
+            choice.append_content(
+                f"![Image](data:image/{image_extension};base64,{image_base64})"
+            )
+
+            # Add an attachment with the same user image, but with using url
```
Comment (adubovik) [inline examples/image-echo/app.py]: `but using url`

Comment (vladisavvv) [reply examples/image-echo/app.py]: fixed

File: examples/langchain-rag/app.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            embeddings = CacheBackedEmbeddings.from_bytes_store(
+                openai_embedding,
+                embedding_store,
+                namespace=openai_embedding.model,
+            )
+
+            if len(request.messages) == 1:
+                # Create the download stage to show to the user the active process.
+                # After the loading is complete, the stage will auto finished.
+                with choice.create_stage("Download the resource"):
```
Comment (adubovik) [inline examples/langchain-rag/app.py]: `Downloading the resource` as a continious process looks better as a stage name. Same for other stages.

`Download the resource` sounds like a command: `You! Download the resource`.

File: examples/langchain-rag/app.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                    "The resource is loaded. Feel free to ask questions about it."
+                )
+            else:
+                documents = loader.load()
+                texts = text_splitter.split_documents(documents)
+                docsearch = Chroma.from_documents(
+                    texts, embeddings, collection_name=collection_name
+                )
+
+                # DIAL Api Key and authorization headers will be detected automaticly and taken from the original application request
```
Comment (adubovik) [inline examples/langchain-rag/app.py]: Typo `automatically`

**Review by adubovik — APPROVED** [review]

**Pull request merged**